### DISCUSSION TITLE
【求助】我的绳网为什么很多图片都显示不出来

### DISCUSSION THREAD

**Discussion by Rope354864** [share121/inter-knot#2765] [其他]
![Uploading image.png…]()

**Comment by OttoApocalypse01** [comment]
可能是卡顿造成的，试着打开steam++加速一下github然后重进绳网<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3996afc68a13bfc5b5718af31e610f43_6807527097924316704.png" alt="罗刹 送花" width="75" >

**Reply by Rope354864** [reply]
谢谢，已经解决了

**Comment by LeiMoo** [comment]
因为走的github的服务，而它的服务器基本在国外，国内没加速访问就很慢，有的地方dns被污染的话根本进不来，电脑有steam++、steamcommunity_302（dogfight360大佬的，你可以去了解一下他的其他工具也能加速github），手机端的话找v@p@n有免费也有付费的

**Reply by Rope354864** [reply]
感谢回复（：
